libkbfs/bserver_remote.go

```diff
@@ -376,8 +376,15 @@ func (b *BlockServerRemote) Get(ctx context.Context, tlfID tlf.ID, id kbfsblock.
 	context kbfsblock.Context) (
 	buf []byte, serverHalf kbfscrypto.BlockCryptKeyServerHalf, err error) {
 	ctx = rpc.WithFireNow(ctx)
+	var res keybase1.GetBlockRes
 	b.log.LazyTrace(ctx, "BServer: Get %s", id)
+
+	// Once the block has been retrieved, cache it.
 	defer func() {
+		// But don't cache it if it's archived data.
+		if res.Status == keybase1.BlockStatus_ARCHIVED {
+			return
+		}
```

Comment:
Should this be inside the `else` below and right after `CDebugf` so that we still get the logs?

Reply:
I agree we still want to have the log in this case.